src/broker/cas_error.c

```diff
@@ -52,6 +52,8 @@
 #include "broker_filename.h"
 #include "cas_sql_log2.h"
 
+#define CUBRID_SQL_LOG_TRACE        "CUBRID_SQL_LOG_TRACE"
```

Comment:
It seems that the env name could be mistaken for users to trace sql.
How about changing the name to CUBRID_CAS_ERR_TRACE ?

Reply:
Thanks for the advice.
Changed it to CUBRID_CAS_ERR_TRACE.